smilofoundation/go-smilo
# Update build, version, travis, enable tendermint tests, update deps, update lint, update go version #29

* add focal deb build
* update lint to golangci-lint-1.22.2
* update go to go1.13.8
* add timeout flag to build cmd
* update ci-notes.md
* enable all minus one tendermint_test.go
* bump version.go to 1.9.2.4
* update gocigar version and enable osx 386 build
* bump travis osx image to xcode11.3
* add travis build step for tendermint tests
* update README.md adding smiloutils and extradata cmds and some generic Consensus section
* fix Close method of tendermint/backend/backend.go 
* reduce a bit sizes of the logs
* add stages on travis build and add TENDERMINT_SLOW in allow_failures
* add close to node.go
* add check for worker.go for empty block avoid nil pointer
* skip comparing hashes that are empty on tendermint_test.go
* add sport.Close after sport.Stop inside the miner worker.go
* move all slow tendermint to non blocking build
* fix handler.go to avoid closing backend on stop
